packages/state/src/recs/index.ts

```
@@ -193,6 +199,8 @@ export const getEvents = async <S extends Schema>(
             offset += limit;
         }
     }
+
+    callback && callback();
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)